src/fetchTopLanguages.js

```diff
@@ -72,11 +71,12 @@ async function fetchTopLanguages(username) {
       };
     }, {});
 
-  const topLangs = Object.keys(repoNodes)
+  const topLangs = Object.values(repoNodes)
+    .sort((a, b) => b.size - a.size)
```

Comment:
We are already doing this on renderTopLanguages

https://github.com/anuraghazra/github-readme-stats/blob/a4486d032705f804965f22ebea4ed268e2af49a0/src/renderTopLanguages.js#L90

In #247 the sorting should happen while calculating the reduced language size not when we are slicing the languages.

Reply:
The calculated repoNodes is an object, should I sort the object?

Reply:
I think what you are doing is fine, because we are already doing Object.values()

Reply:
Yes, please merge it